pkg/lib/cockpit-components-shutdown.jsx

```diff
@@ -31,7 +31,7 @@ import { DatePicker } from "@patternfly/react-core/dist/esm/components/DatePicke
 import { TimePicker } from "@patternfly/react-core/dist/esm/components/TimePicker/index.js";
 
 import { ServerTime } from 'serverTime.js';
-import * as timeformat from "timeformat.js";
+import * as timeformat from "timeformat";
```

Comment:
We added a rule to Cockpit Files to forbid file extension on imports.....

Topic for another PR :)